### DISCUSSION TITLE
Solicitação de 1password para a PyNE

### DISCUSSION THREAD

**Discussion by itepifanio** [apyb/comunidade#130] [Geral]
1password tem um [programa para instituições open source](https://github.com/1Password/for-open-source), eu não tenho certeza se conseguimos criar uma conta para a APyB e criar vaults separados por eventos regionais com restrições de acesso, se isso for possível seria o cenário ideal, mas caso contrário queremos solicitar o contato pra criação de uma conta apenas para a PyNE.

A ideia de utilizar o 1password é ter uma facilidade maior de utilizar a autenticação da conta do google, algo que é limitado no bitwarden.

**Comment by ericof** [comment]
Adendo: A gestão do permissionamento ficaria a cargo da APyB, que criaria grupos (Eu posso auxiliar nesse processo pois temos a mesma solução na Plone Foundation)

**Comment by cecivieira** [comment]
@naanadr

**Comment by naanadr** [comment]
@itepifanio eu criei no começo do ano uma conta no 1password para a PyBR (https://github.com/apyb/tarefas/issues/520), porque o evento é gerido pela APyB, mas a nossa conta tem limite de 10 usuários ([aqui](https://1password.com/product/teams-small-business-password-manager?unleash_session_id=427745736)) 😢 . Por isso seria preciso submeter no [1Password/for-open-source/](https://github.com/1Password/for-open-source/) um PR solicitando upgrade gratuito de um domínio para a Python Nordeste.

Como cada evento regional faz o gerenciamento de forma independente, acho que vocês poderiam ficar responsáveis por criar o 1password e solicitar no repositório pelo upgrade. Assim vocês conseguem adicionar e remover as pessoas do vault mais facilmente e não vão precisar recorrer a APyB sempre que a organização mudar.

Algumas dicas para ajudar nesse gerenciamento:
- Sempre ter pelo menos duas contas como owner do domínio;
- A pessoa big kahuna sempre deve ser um dos owner do domínio e do vault.

cc @cecivieira

**Comment by cecivieira** [comment]
@cecivieira entendi.
Eu sou a owner do cofre da Python Nordeste.

Vou ficar repsonsável por criar o 1Passwor, então.

**Comment by cecivieira** [comment]
@itepifanio tu poderia tomar conta disso? fazer a conta e pedir a gratuidade?
ainda tás logado na conta da python nordeste? essa conta deve ser a proprietária do cofre.

Quando tiver tudo certinho, avisa aqui, por favor. Assim, poderei dar andamento ao que mais precisar.

**Reply by itepifanio** [reply]
Estou logado na conta da PyNE sim, pode deixar que vou dar andamento a solicitação.

**Reply by naanadr** [reply]
@itepifanio depois seria legal colocar tbm como proprietário o email da pessoa big Kahuna. Pq alguém tem que ficar de backup para conseguir gerenciar o cofre caso algum dos emails perca acesso. 

Vcs conseguem transferir a propriedade de uma pessoa para a outra, é bem customizável o domínio. 

E não esqueçam de armazenar a senha e a chave de acesso do email da PyNe em um lugar seguro, se não vcs vão perder o acesso a conta 😅

**Reply by itepifanio** [reply]
@naanadr me coloquei como owner, já convidei a Cecilia e as novas big Kahunas, quando terminarmos o post-mortem da edição 2024 o pessoal tira meu acesso.

A aplicação ficou aqui: https://github.com/1Password/for-open-source/issues/1007

**Reply by itepifanio** [reply]
@cecivieira @naanadr tudo certo com a nossa conta 1Password

@cecivieira você tem acesso direto ao Bitwarden? Caso tenha exporta por favor o cofre do Bitwarden e importa via 1Password (https://support.1password.com/import-bitwarden/), meu acesso não permite isso. Uma vez feita a transferência do cofre podemos fechar essa discussão e [essa outra issue](https://github.com/pythonNordeste/organizacao-pyne-2025/issues/2#issuecomment-2305466798)

Se der já coloca a autenticação de 2 fatores do gmail na conta por favor.

**Reply by cecivieira** [reply]
@itepifanio fiz a migracao das senhas para o 1Password.
Tu poderia configurar o código temporário para autenticação de 2 fatores(nao lembro como chama), caso ainda não esteja?

**Reply by itepifanio** [reply]
@cecivieira tudo certo, nada errado.

Autenticação de dois fatores já foi adicionada tanto pra conta google quanto pro instagram. Flickr não tem autenticação de dois fatores, mas o restante dos apps vou adicionando a autenticação aos poucos.

A conta google ainda pede acesso inicial a um celular readmi note 11, mas selecionando "tentar de outra forma", selecionando a opção de "código do google authenticator" e colocando a chave do 1password dá pra acessar.

Achei que tinha tirado o celular das opções de entrada, mas não parece ter funcionado. Vou verificar isso mais tarde.

**Reply by cecivieira** [reply]
@itepifanio massa!!!!
